src/words/ast/INodeCreateLocalVariable.java

```diff
@@ -0,0 +1,28 @@
+package words.ast;
+
+import words.environment.Variable;
+import words.environment.*;
+import words.exceptions.*;
+
+public class INodeCreateLocalVariable extends INode {
+	public INodeCreateLocalVariable(Object... children) {
+		super(children);
+	}
+
+	@Override
+	public ASTValue eval(Environment environment) throws WordsRuntimeException {
+		ASTValue varName = children.get(0).eval(environment);
+		ASTValue rawValue = children.get(1).eval(environment);
+		Variable value = null;
+		
+		if (rawValue.type == ASTValue.Type.NUM) {
+			value = new Variable(rawValue.numValue);
+		} else if (rawValue.type == ASTValue.Type.STRING) {
+			value = new Variable(rawValue.stringValue);
+		}
+		
```

Comment:
What about handling the case where the ASTValue is NOTHING?  It should be legal to assign a variable to be nothing, as in, "x is nothing.".

Also, is it legal to assign a variable to an object?  For example:
  Fred is a person.
  X is Fred.
(If that's legal, garbage collection becomes harder, so it's probably not legal.  But if it's not legal, then the code here will silently do nothing, whereas I think it should give a runtime exception to the user.)